packages/skeleton-svelte/src/components/Toast/ToastProvider.svelte

```diff
@@ -1,133 +1,76 @@
 <script lang="ts">
-	import { setContext } from 'svelte';
-	import { flip } from 'svelte/animate';
-	import { fade } from 'svelte/transition';
-
-	import type { PlacementStyles, Toast, ToastProviderProps } from './types.js';
+	import { normalizeProps, useMachine } from '@zag-js/svelte';
+	import * as toast from '@zag-js/toast';
+	import Toast from './Toast.svelte';
+	import type { ToastProviderProps } from './types.js';
+	import { setToastContext } from './context.js';
 
 	const {
-		placement = 'bottom-end',
-		offset = '16px',
-		dismissLabel = '',
-		// Group
-		base = 'fixed flex flex-col items-end',
-		gap = 'gap-4',
-		zIndex = 'z-[888]',
-		classes = '',
 		// Toast
-		toastBase = 'card py-2 px-3 grid grid-cols-[1fr_auto] items-center',
-		toastPadding = 'py-2 px-3',
-		toastGap = 'gap-4',
-		toastShadow = 'shadow-xl',
-		toastClasses = '',
+		base = 'flex justify-between items-center gap-3',
+		width = 'min-w-xs',
+		padding = 'p-3',
+		rounded = 'rounded-container',
+		classes = '',
 		// Message
-		messageBase = 'grid grid-cols-1 max-w-xs text-xs',
-		messageTitle = 'font-bold',
-		messageDescription = '',
+		messageBase = 'grid',
 		messageClasses = '',
+		// Title
+		titleBase = 'font-semibold',
+		titleClasses = '',
+		// Description
+		descriptionBase = 'text-sm',
+		descriptionClasses = '',
 		// Dismiss Button
-		buttonDismissBase = 'btn-icon btn-icon-sm text-base',
-		buttonDimissPreset = '',
-		buttonDismissHover = 'hover:preset-tonal',
+		buttonDismissBase = 'btn-icon hover:preset-tonal',
 		buttonDismissClasses = '',
 		// State
-		stateInfo = 'preset-filled',
-		stateError = 'preset-filled-error-500',
+		stateInfo = 'preset-outlined-surface-200-800 preset-filled-surface-50-950',
```

Comment:
FYI: Let's just use Presets for now, we'll reconsider our options in the next major release.